layers/+source-control/git/README.org

```diff
@@ -136,6 +159,48 @@ To enable it you have to add the following lines to your
 #+END_SRC
 
 ** Forge
+
+Magit Forge can view and create issues & pull requests with forges
+(e.g. GitHub, GitLab)
+
+Magit Forge requires a the username for the respective forge and will prompt for a
+value if not defined in  =~/.gitconfig=
+Or define your forge identity using the following command in a terminal window
+
+For GitHub:
+
+#+begin_src shell
+  git config --global github.name "username"
+#+end_src
+
+For GitLab:
+
+#+begin_src shell
+  git config --global gitlab.name "username"
+#+end_src
+
+See the official [[https://magit.vc/manual/forge/Getting-Started.html#Getting-Started][Magit Forge]] and [[https://magit.vc/manual/ghub/Getting-Started.html][GHub Getting Started]] for general guides or follow
+a community written [[hhttps://practical.li/spacemacs/source-control/forge-configuration.html][Spacemacs specific guide to configuring Magit Forge]].
```

Comment:
```suggestion
a community written [[https://practical.li/spacemacs/source-control/forge-configuration.html][Spacemacs specific guide to configuring Magit Forge]].
```

Reply:
fixed

Reply:
As far as I can tell, that one is actually not fixed. (Forgot to stage?)